src/pages/docs/infrastructure/workers/dynamic-worker-pools.md

```diff
@@ -88,23 +88,25 @@ Ubuntu 18.04 images are no longer available as of 3 April 2023. Please refer to
 
 ### Windows Server Core 2019
 
-This is the default for the Windows operating system, referenced as `Windows (default)`.
+:::div{.warning}
+Windows 2019 is currently the `Windows (default)` image. Windows 2019 images will be deprecated on 9 January 2024. You are advised to test your deployment processes with our Windows 2022 images. Please refer to [Windows 2019 end-of-life](/docs/infrastructure/workers/dynamic-worker-pools/windows-2019-end-of-life.md) for further details.
+:::
 
 Each `Windows Server Core 2019` worker is provisioned with a baseline of tools including (but not limited to):
 
 - .NET Core (2.1, 3.1)
 - .NET Framework 3.5
 - .NET Framework 4.8
-- AWS IAM Authenticator (0.5.1)
+- AWS IAM Authenticator (0.5.3)
 - Chocolatey (latest)
-- Docker (19.03.5)
+- Docker (latest)
 - Helm (2.9.1)
-- Kubectl (1.16.10)
+- Kubectl (multiple versions)
 - Microsoft Service Fabric (6.1.480.9494)
 - Microsoft Service Fabric SDK (3.0.480)
 - Nuget CLI (latest)
 - Octopus Client (latest)
-- Pip (20.1.1)
+- Pip (latest)
```

Comment:
we upgrade it: https://github.com/OctopusDeploy/DynamicWorkerVmImages/blob/master/Win2019/scripts/upgrade-pip.ps1#L3